<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Publications</title>
<meta name="generator" content="Org Mode" />
<link rel="stylesheet" type="text/css" href="css/custom.css"/>
<script type="text/javascript" href="css/custom.js"/>

        <style>
        /* From: https://endlessparentheses.com/public/css/endless.css */
        /* See also: https://meta.superuser.com/questions/4788/css-for-the-new-kbd-style */
        kbd
        {
          -moz-border-radius: 6px;
          -moz-box-shadow: 0 1px 0 rgba(0,0,0,0.2),0 0 0 2px #fff inset;
          -webkit-border-radius: 6px;
          -webkit-box-shadow: 0 1px 0 rgba(0,0,0,0.2),0 0 0 2px #fff inset;
          background-color: #f7f7f7;
          border: 1px solid #ccc;
          border-radius: 6px;
          box-shadow: 0 1px 0 rgba(0,0,0,0.2),0 0 0 2px #fff inset;
          color: #333;
          display: inline-block;
          font-family: 'Droid Sans Mono', monospace;
          font-size: 80%;
          font-weight: normal;
          line-height: inherit;
          margin: 0 .1em;
          padding: .08em .4em;
          text-shadow: 0 1px 0 #fff;
          word-spacing: -4px;
        
          box-shadow: 2px 2px 2px #222; /* MA: An extra I've added. */
        }
        </style>
        <link rel="stylesheet" type="text/css" href="https://alhassy.github.io/org-special-block-extras/tooltipster/dist/css/tooltipster.bundle.min.css"/>
        
        <link rel="stylesheet" type="text/css" href="https://alhassy.github.io/org-special-block-extras/tooltipster/dist/css/plugins/tooltipster/sideTip/themes/tooltipster-sideTip-punk.min.css" />
        
        <script type="text/javascript">
            if (typeof jQuery == 'undefined') {
                document.write(unescape('%3Cscript src="https://code.jquery.com/jquery-1.10.0.min.js"%3E%3C/script%3E'));
            }
        </script>
        
         <script type="text/javascript"            src="https://alhassy.github.io/org-special-block-extras/tooltipster/dist/js/tooltipster.bundle.min.js"></script>
        
          <script>
                 $(document).ready(function() {
                     $('.tooltip').tooltipster({
                         theme: 'tooltipster-punk',
                         contentAsHTML: true,
                         animation: 'grow',
                         delay: [100,500],
                         // trigger: 'click'
                         trigger: 'custom',
                         triggerOpen: {
                             mouseenter: true
                         },
                         triggerClose: {
                             originClick: true,
                             scroll: true
                         }
         });
                 });
             </script>
        
        <style>
           abbr {color: red;}
        
           .tooltip { border-bottom: 1px dotted #000;
                      color:red;
                      text-decoration: none;}
        </style>
</head>
<body>
<div id="preamble" class="status">
<nav class="navMenu"><a href="index.html">Home</a><a href="publications.html">Publications</a></nav>
</div>
<div id="content" class="content">
<h1 class="title">Publications</h1>


<div id="outline-container-org9202a56" class="outline-2">
<h2 id="org9202a56">Thesis</h2>
<div class="outline-text-2" id="text-org9202a56">
<div class="article"><div class="article-title">À la recherche de l'échafaudage parfait : efficace, de qualité et garanti (in french)</div><div class="author">Tom Davot</div><div class="journal">Thesis</div><div class="year">2020</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://theses.hal.science/tel-03158090" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://theses.hal.science/tel-03158090" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
Sequencing is a process in biology that determines the order of nucleotides in the DNA. It produces a set of fragments, called reads, in which the genetic information is known. Unfortunatly, the genomic sequence is decomposed in small pieces. In order to analyse it, it is necessary to reconstruct it using a number of computer processes. In this thesis, we studied two mathematical problems arising from this sequencing: the scaffolding and the linearization.The scaffolding is a process that takes place after the reads assembly into larger subsequences called contigs. It consists in the search of paths and cycles in a particular graph called scaffold graph. These paths and cycles represent the linear and circular chromosomes of the organism whose DNA has been sequenced. The linearization is a problem related to the scaffolding. When we take into account that contigs may appear several times in the genomic sequence, some ambiguities can arise. If this ambiguities are not deleted, then a chimeric sequence may be produced by the scaffolding. To solve this problem, a solution computed by the scaffolding should be wisely deteriorated. In any case, both problems can be modelized as optimization problems in a graph.In this document, we study both problems focusing on three aspects. The first aspect consists in the study of the complexity of these problems. The second aspect consists in the development of algorithms, exact or approximate, to solve these problems. Finally, the last aspect consists in implementing and testing these algorithms to look at their behaviors on real instances.
</details></div>
</div>
</div>


<div id="outline-container-org91effd8" class="outline-2">
<h2 id="org91effd8">Journal</h2>
<div class="outline-text-2" id="text-org91effd8">
</div>
<div id="outline-container-org74f8733" class="outline-3">
<h3 id="org74f8733">2023</h3>
<div class="outline-text-3" id="text-org74f8733">
<div class="article"><div class="article-title">On the Shared Transportation Problem: Computational Hardness and Exact Approach</div><div class="author">Tom Davot, Rodolphe Giroudeau and Jean-Claude König</div><div class="journal">International Journal of Foundations of Computer Science</div><div class="year">2023</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1142/S0129054123410058" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/370645383_On_the_Shared_Transportation_Problem_Computational_Hardness_and_Exact_Approach" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
In our modern societies, a certain number of people do not own a car, by choice or by obligation. For some trips, there is no or few alternatives to the car. One way to make these trips possible for these people is to be transported by others who have already planned their trips. We propose to model this problem using as path-finding problem in a list edge-colored graph. This problem is a generalization of the s-t-path problem, studied by Böhmová et al. We consider two optimization functions: minimizing the number of color changes and minimizing the number of colors. We study for the previous problems, the classic complexity (polynomial-case, NP-completeness, hardness of approximation) and parameter complexity (W[2]-hardness) even in restricted cases. We also propose a lower bound for exact algorithm. On the positive side we provide a polynomial-time approximation algorithm and a FPT algorithm.
</details></div>
</div>
</div>

<div id="outline-container-orgcf26ef6" class="outline-3">
<h3 id="orgcf26ef6">2022</h3>
<div class="outline-text-3" id="text-orgcf26ef6">
<div class="article"><div class="article-title">On a greedy approach for genome scaffolding</div><div class="author">Tom Davot, Annie Chateau, Rohan Fossé, Rodolphe Giroudeau and Mathias Weller</div><div class="journal">Algorithms for Molecular Biology</div><div class="year">2022</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1186/s13015-022-00223-x" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/364890682_On_a_greedy_approach_for_genome_scaffolding" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
*Background*
Scaffolding is a bioinformatics problem aimed at completing the contig assembly process by determining the relative position and orientation of these contigs. It can be seen as a paths and cycles cover problem of a particular graph called the “scaffold graph”.

*Results*
We provide some NP-hardness and inapproximability results on this problem. We also adapt a greedy approximation algorithm on complete graphs so that it works on a special class aiming to be close to real instances. The described algorithm is the first polynomial-time approximation algorithm designed for this problem on non-complete graphs.

*Conclusion*
Tests on a set of simulated instances show that our algorithm provides better results than the version on complete graphs.
</details></div>
</div>
</div>

<div id="outline-container-orgdad25ed" class="outline-3">
<h3 id="orgdad25ed">2021</h3>
<div class="outline-text-3" id="text-orgdad25ed">
<div class="article"><div class="article-title">Producing Genomic Sequences after Genome Scaffolding with Ambiguous Paths: Complexity, Approximation and Lower Bounds</div><div class="author">Tom Davot, Annie Chateau, Rodolphe Giroudeau, Mathias Weller and Dorine Tabary</div><div class="journal">Algorithmica</div><div class="year">2021</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/s00453-021-00819-6" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/351281260_Producing_Genomic_Sequences_after_Genome_Scaffolding_with_Ambiguous_Paths_Complexity_Approximation_and_Lower_Bounds" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
Scaffolding is the final step in assembling Next Generation Sequencing data, in which pre-assembled contiguous regions (”contigs”) are oriented and ordered using information that links them (for example, mapping of paired-end reads). As the genome of some species is highly repetitive, we allow placing some contigs multiple times, thereby generalizing established computational models for this problem. We study the subsequent problems induced by the translation of solutions of the model back to actual sequences, proposing models and analyzing the complexity of the resulting computational problems. We find both polynomial-time and NP-hard special cases like planarity or bounded degree. Finally, we propose two polynomial-time approximation algorithms according to cut/weight score.
</details></div>

<div class="article"><div class="article-title">Efficient assembly consensus algorithms for divergent contig sets</div><div class="author">Annie Chateau, Tom Davot and Manuel Laffond</div><div class="journal">Computational Biology and Chemistry</div><div class="year">2021</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1016/j.compbiolchem.2021.107516" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/351805198_Efficient_Assembly_Consensus_Algorithms_for_Divergent_Contig_Sets" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
Assembly is a fundamental task in genome sequencing, and many assemblers have been made available in the last decade. Because of the wide range of possible choices, it can be hard to determine which tool or parameter to use for a specific genome sequencing project. In this paper, we propose a consensus approach that takes the best parts of several contigs datasets produced by different methods, and combines them into a better assembly. This amounts to orienting and ordering sets of contigs, which can be viewed as an optimization problem where the aim is to find an alignment of two fragmented strings that maximizes an arbitrary scoring function between matched characters. In this work, we investigate the computational complexity of this problem. We first show that it is NP-hard, even in an alphabet with only two symbols and with all scores being either 0 or 1. On the positive side, we propose an efficient, quadratic time algorithm that achieves approximation factor 3.
</details></div>
</div>
</div>

<div id="outline-container-org316999a" class="outline-3">
<h3 id="org316999a">2019</h3>
<div class="outline-text-3" id="text-org316999a">
<div class="article"><div class="article-title">A general decomposition theory for the 1-2-3 Conjecture and locally irregular decompositions</div><div class="author">Olivier Baudon, Julien Bensmail, Tom Davot, Hervé Hocquard, Jakub Przybylo, Mohammed Senhaji, Éric Sopena and Mariusz Wozniak</div><div class="journal">Discrete Mathematics & Theoretical Computer Science</div><div class="year">2019</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.23638/DMTCS-21-1-2" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/332168100_A_general_decomposition_theory_for_the_1-2-3_Conjecture_and_locally_irregular_decompositions" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
How can one distinguish the adjacent vertices of a graph through an edge-weighting? In the last decades, this question has been attracting increasing attention, which resulted in the active field of distinguishing labellings. One of its most popular problems is the one where neighbours must be distinguishable via their incident sums of weights. An edge-weighting verifying this is said neighbour-sum-distinguishing. The popularity of this notion arises from two reasons. A first one is that designing a neighbour-sum-distinguishing edge-weighting showed up to be equivalent to turning a simple graph into a locally irregular (i.e., without neighbours with the same degree) multigraph by adding parallel edges, which is motivated by the concept of irregularity in graphs. Another source of popularity is probably the influence of the famous 1-2-3 Conjecture, which claims that such weightings with weights in 1,2,3 exist for graphs with no isolated edge. The 1-2-3 Conjecture has recently been investigated from a decompositional angle, via so-called locally irregular decompositions, which are edge-partitions into locally irregular subgraphs. Through several recent studies, it was shown that this concept is quite related to the 1-2-3 Conjecture. However, the full connexion between all those concepts was not clear. In this work, we propose an approach that generalizes all concepts above, involving coloured weights and sums. As a consequence, we get another interpretation of several existing results related to the 1-2-3 Conjecture. We also come up with new related conjectures, to which we give some support.
</details></div>
</div>
</div>
</div>

<div id="outline-container-org258dffc" class="outline-2">
<h2 id="org258dffc">Conference</h2>
<div class="outline-text-2" id="text-org258dffc">
</div>
<div id="outline-container-orgd6428e1" class="outline-3">
<h3 id="orgd6428e1">2023</h3>
<div class="outline-text-3" id="text-orgd6428e1">
<div class="article"><div class="article-title">Degreewidth: a New Parameter for Solving Problems on Tournaments</div><div class="author">Tom Davot, Lucas Isenmann and Sanjukta Roy and Jocelyn Thiebaut</div><div class="journal">WG, Fribourg (Switzerland)</div><div class="year">2023</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://arxiv.org/abs/2212.06007" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/366212637_Degreewidth_a_New_Parameter_for_Solving_Problems_on_Tournaments" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
In the paper, we define a new parameter for tournaments called degreewidth which can be seen as a measure of how far is the tournament from being acyclic. The degreewidth of a tournament T denoted by \Delta(T) is the minimum value k for which we can find an ordering \langle v_1, \dots, v_n \rangle of the vertices of T such that every vertex is incident to at most k backward arcs (\textit{i.e.</details></div>
</div>
</div>



<div id="outline-container-orgfd7c3e5" class="outline-3">
<h3 id="orgfd7c3e5">2022</h3>
<div class="outline-text-3" id="text-orgfd7c3e5">
<div class="article"><div class="article-title">K-Partitioning with Imprecise Probabilistic Edges</div><div class="author">Tom Davot, Sébastien Destercke and David Savourey</div><div class="journal">SMPS, Valladolid (Spain)</div><div class="year">2022</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/978-3-031-15509-3_12" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/362929021_K-Partitioning_with_Imprecise_Probabilistic_Edges" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
Partitioning a set of elements into disjoint subsets is a common problem in unsupervised learning (clustering) as well as in networks (e.g., social, ecological) where one wants to find heterogeneous subgroups such that the elements within each subgroup are homogeneous. In this paper, we are concerned with the case where we imprecisely know the probability that two elements should belong to the same partition, and where we want to search the set of most probable partitions. We study the corresponding algorithmic problem on graphs, showing that it is difficult, and propose heuristic procedures that we test on data sets.
</details></div>
</div>
</div>


<div id="outline-container-org31ba1e5" class="outline-3">
<h3 id="org31ba1e5">2021</h3>
<div class="outline-text-3" id="text-org31ba1e5">
<div class="article"><div class="article-title">Complexity and Approximation Results on the shared transportation problem</div><div class="author">Tom Davot, Rodolphe Giroudeau and Jean-Claude König</div><div class="journal">COCOA, Tianjin (China)</div><div class="year">2021</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/978-3-030-92681-6_12" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/356956999_Complexity_and_Approximation_Results_on_the_Shared_Transportation_Problem" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
In our modern societies, a certain number of people do not own a car, by choice or by obligation. For some trips, there is no or few alternatives to the car. One way to make these trips possible for these people is to be transported by others who have already planned their trips. We propose to model this problem using as path-finding problem in a list edge-colored graph. This problem is a generalization of the s-t-path problem, studied by Böhmová et al. We study two optimization functions: minimizing the number of color changes and minimizing the number of colors. We study the complexity and the approximation of this problem. We show the existence of polynomial cases. We show that this problem is NP-complete and hard to approximate, even in restricted cases. Finally, we provide an approximation algorithm. 
</details></div>


<div class="article"><div class="article-title">On the approximation hardness of geodetic set and its variants</div><div class="author">Tom Davot, Lucas Isenmann and Jocelyn Thiebaut:</div><div class="journal">COCOON, Taina (Taiwan)</div><div class="year">2021</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/978-3-030-89543-3_7" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/356503645_On_the_approximation_hardness_of_geodetic_set_and_its_variants" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
Given a graph, a geodetic set (resp. edge geodetic set) is a subset of its vertices such that every vertex (resp. edge) of the graph is on a shortest path between two vertices of the subset. A strong geodetic set is a subset S of vertices and a choice of a shortest path for every pair of vertices of S such that every vertex is on one of these shortest paths. The geodetic number (resp. edge geodetic number) of a graph is the minimum size of a geodetic set (resp. edge geodetic set) and the strong geodetic number is the minimum size of a strong geodetic set. We first prove that, given a subset of vertices, it is NP-hard to determine whether it is a strong geodesic set. Therefore, it seems natural to study the problem of maximizing the number of covered vertices by a choice of a shortest path for every pair of a provided subset of vertices. We provide a tight 2-approximation algorithm to solve this problem. Then, we show that there is no 781/780-polynomial-time approximation algorithm for edge geodetic number and strong geodetic number on subcubic bipartite graphs with arbitrarily high girth. We also prove that geodetic number and edge geodetic number are both LOG-APX-hard, even on subcubic bipartite graphs with arbitrarily high girth. Finally, we disprove a conjecture of Iršiš and Konvalinka by proving that the strong geodetic number can be computed in polynomial time in complete multipartite graphs.
</details></div>
</div>
</div>

<div id="outline-container-org30aa2dc" class="outline-3">
<h3 id="org30aa2dc">2020</h3>
<div class="outline-text-3" id="text-org30aa2dc">
<div class="article"><div class="article-title">Linearizing Genomes: Exact Methods and Local Search</div><div class="author">Tom Davot, Annie Chateau, Rodolphe Giroudeau and Mathias Weller</div><div class="journal">SOFSEM, Limassol (Cyprus)</div><div class="year">2020</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/978-3-030-38919-2_41" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/338638955_Linearizing_Genomes_Exact_Methods_and_Local_Search" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
In this article, we address the problem of genome linearization from the perspective of Polynomial Local Search, a complexity class related to finding local optima. We prove that the linearization problem, with a neighborhood structure, the neighbor slide, is PLS-complete. On the positive side, we develop two exact methods, one using tree decompositions with an efficient dynamic programming, the other using an integer linear programming. Finally, we compare them on real instances.
</details></div>
</div>
</div>

<div id="outline-container-org64ae7fd" class="outline-3">
<h3 id="org64ae7fd">2019</h3>
<div class="outline-text-3" id="text-org64ae7fd">
<div class="article"><div class="article-title">New Polynomial-Time Algorithm Around the Scaffolding Problem</div><div class="author">Tom Davot, Annie Chateau, Rodolphe Giroudeau and Mathias Weller</div><div class="journal">ALCOB, Berkely (USA)</div><div class="year">2019</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/978-3-030-18174-1_2" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/333224841_New_Polynomial-Time_Algorithm_Around_the_Scaffolding_Problem" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
We describe in this paper an approximation algorithm for the scaffolding problem, which is part of genome inference in bioinformatics. The aim of the problem is to find a maximum weighted collection of disjoint alternating cycles and paths covering a particular graph called scaffold graph. The problem is known to be NP-complete, and we describe further result concerning a special class of graphs aiming to be close to real instances. The described algorithm is the first polynomial-time approximation algorithm designed for this problem on non-complete graphs.
</details></div>
</div>
</div>

<div id="outline-container-org8c21e77" class="outline-3">
<h3 id="org8c21e77">2018</h3>
<div class="outline-text-3" id="text-org8c21e77">
<div class="article"><div class="article-title">New Results About the Linearization of Scaffolds Sharing Repeated Contigs</div><div class="author">Dorine Tabary, Tom Davot, Mathias Weller, Annie Chateau and Rodolphe Giroudeau</div><div class="journal">COCOA, Atlanta (USA)</div><div class="year">2018</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/978-3-030-04651-4_7" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/328973928_New_Results_About_the_Linearization_of_Scaffolds_Sharing_Repeated_Contigs_12th_International_Conference_COCOA_2018_Atlanta_GA_USA_December_15-17_2018_Proceedings" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
Solutions to genome scaffolding problems can be represented as paths and cycles in a “solution graph”. However, when working with repetitions, such solution graphs may contain branchings and, thus, they may not be uniquely convertible into sequences. Having introduced various ways of extracting the unique parts of such solutions, we extend previously known NP-hardness results to the case that the solution graph is planar, bipartite, and subcubic, and show that there is no PTAS in this case.
</details></div>

<div class="article"><div class="article-title">On the Hardness of Approximating Linearization of Scaffolds Sharing Repeated Contigs</div><div class="author">Tom Davot, Annie Chateau, Rodolphe Giroudeau and Mathias Weller:</div><div class="journal">RECOMB-CG, Sherbrook (Canada)</div><div class="year">2018</div><details class="abstract"> <summary class="abstract-title"><strong>Abstract</strong><a href="https://doi.org/10.1007/978-3-030-00834-5_5" target="_blank"><img src="img/doi.png" alt="doi logo" style="width:5%; margin: 0px 10px"></a><a href="https://www.researchgate.net/publication/327508812_On_the_Hardness_of_Approximating_Linearization_of_Scaffolds_Sharing_Repeated_Contigs_16th_International_Conference_RECOMB-CG_2018_Magog-Orford_QC_Canada_October_9-12_2018_Proceedings" target="_blank"><img src="img/rg.svg" alt="rg logo" style="width:5%"></a></summary><hr>
Solutions to genome scaffolding problems can be represented as paths and cycles in a “solution graph”. However, when working with repetitions, such solution graph may contain branchings and they may not be uniquely convertible into sequences. Having introduced, in a previous work, various ways of extracting the unique parts of such solutions, we extend previously known NP-hardness results to the case that the solution graph is planar, bipartite, and subcubic, and show the APX-completeness in this case. We also provide some practical tests.
</details></div>
</div>
</div>
</div>
</div>
</body>
</html>
